### DISCUSSION TITLE
Empty filters return different values

### DISCUSSION THREAD

**Discussion by bentodvictor** [grafana/grafana#53393] [Alerting]
Hello everyone.
When Grafana filters is empty, the value of these field have two values randomly:

1. Filters:
![Screenshot from 2022-08-08 09-46-45](https://user-images.githubusercontent.com/32740824/183421492-43977202-ff24-4dee-8e76-8d7868e420da.png)

2. All filters values are captured using Flux syntax:
![Screenshot from 2022-08-08 09-47-08](https://user-images.githubusercontent.com/32740824/183421505-23ccfe58-0e5a-4265-9b3f-3bda90c645db.png)

3. The result when filter is empty have two behaviors, somentimes returned as a empty string (""), sometimes is nothing (): 
![Screenshot from 2022-08-08 09-47-17](https://user-images.githubusercontent.com/32740824/183421512-90c8537f-e2ab-409b-959b-e7fdd9ff383c.png)

This is a normal behavior or is a bug? 
Thanks!!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
